src/requireAuth.ts

```diff
@@ -57,7 +57,12 @@ export async function requireAuth(options: any): Promise<void> {
   } else if (user) {
     logger.debug("> authorizing via signed-in user");
   } else {
-    return autoAuth(options, options.authScopes);
+    return autoAuth(options, options.authScopes).catch((e) => {
```

Comment:
my preference would be to do:

```typescript
try {
  await autoAuth();
} catch (e) {
  throw new FirebaseError();
}
```

But I think that's all-in-all a pretty small detail.